corehq/apps/hqwebapp/forms.py

```diff
@@ -111,34 +107,23 @@ def crispy_form_fields(self, context):
 
 
 class AppTranslationsBulkUploadForm(BulkUploadForm):
+    language = forms.CharField(widget=forms.HiddenInput)
     validate = forms.BooleanField(label="Just validate and not update translations", required=False,
                                   initial=False)
-    language = forms.ChoiceField(label=_("Language to validate"), choices=(), required=False)
 
     def crispy_form_fields(self, context):
         crispy_form_fields = super(AppTranslationsBulkUploadForm, self).crispy_form_fields(context)
+        if context.get('can_select_language'):
+            crispy_form_fields.extend([
+                InlineField('language', data_bind="value: lang")
+            ])
```

Comment:
we would need to change this to also have the language field added in case `can_validate_app_translations` is true to support https://github.com/dimagi/commcare-hq/pull/23817 ?